apps/aecore/src/aec_sync.erl

```diff
@@ -106,11 +106,12 @@ compare_ping_objects(Local, Remote) ->
                       _ ->
                           Dl = maps:get(<<"difficulty">>, Local),
                           Dr = maps:get(<<"difficulty">>, Remote),
-                          if Dl > Dr ->
+                          case Dl > Dr of
```

Comment:
This is effectively rewriting an if as a case. I don't see why this fixes a bug in the code. It creates a dependency on aec_sync and if it is just cosmetic, I rather not have it in this PR.